expo/scheduler/models.py

```diff
@@ -845,6 +848,37 @@ def class_contacts(self):
             )
         return info
 
+    def class_contacts2(self):
+        '''
+        To do: reconcile the two class_contacts functions
+        '''
+        allocations = self.resources_allocated.filter(
+            resource__in=Worker.objects.all())
+        info = []
+        for allocation in allocations:
+            try:
+                persona = allocation.resource.item.persona
+                info.append(
+                    (persona.contact_email,
+                     str(self),
+                     allocation.resource.worker.role,
+                     persona.name,
+                     persona.contact.display_name,
+                     persona.contact_phone)
+                )
+            except:
+                profile = allocation.resource.item.profile
```

Comment:
My bet is that this code didn't realize that Acts are also a type of WorkerItem that can get booked?
I'm using a dump of the DB from Friday.